src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -205,6 +205,29 @@ Da nur unpersönliche Zufallscodes übertragen werden, bleibt Ihre Identität un
 "Home_Risk_Status_Check_Body" = "Current data are downloaded and checked. This can take several minutes.";
 "Home_Risk_Status_Check_Button" = "To update";
 
+// Home Thank you card
```

Comment:
wrong comment syntax in translation file: please use /* */ instead of //

Reply:
fixed